lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -44,7 +46,8 @@ export class CdnSiteHostingConstruct extends Construct {
         props.certificateArn,
       ),
       {
-        aliases: [siteDomain],
+        // aliases: [siteDomain],
+        aliases: aliases,
```

Comment:
I think I'd be happier with this:

```suggestion
        aliases: [siteDomain, ...aliases],
```

It makes it clear that the siteDomain is there and that it's the main one.